docs/example_python.md

```diff
@@ -280,7 +291,14 @@ You can try the server here:
 
 Obviously, this is the simplest possible server we could write; but we hope that this gives you a starting point for running your Flask app (or other Python app) via Tilt!
 
-Other examples:
+## Further Reading
+
+### Other sample Python projects:
+- [abc123](https://github.com/windmilleng/abc123) is a mini microservice app with a Python server called `numbers`
+- The [Live Update Tutorial](http://localhost:4001/live_update_tutorial.html) walks you through optimizing a Python server
```

Comment:
:eyes: this tutorial doesn't use python? and also doesn't live on localhost for the reader?

Reply:
er whoops i meant to link to https://blog.tilt.dev/2019/04/02/fast-kubernetes-development-with-live-update.html

Reply:
it might be worth linking to the code repo directly (https://github.com/windmilleng/live_update/tree/master/python)

interestingly, the code repo link in that blog post (https://github.com/windmilleng/live_update/tree/dbentley/initial) is currently broken